src/EnergyPlus/PurchasedAirManager.cc

```diff
@@ -1616,46 +1595,25 @@ void SizePurchasedAir(EnergyPlusData &state, int const PurchAirNum)
                 if (MaxHeatSensCapDes < HVAC::SmallLoad) {
                     MaxHeatSensCapDes = 0.0;
                 }
-                if (IsAutoSize) {
-                    PurchAir(PurchAirNum).MaxHeatSensCap = MaxHeatSensCapDes;
```

Comment:
[src/EnergyPlus/PurchasedAirManager.cc:1619]:(style),[knownConditionTrueFalse],Condition 'IsAutoSize' is always false